### DISCUSSION TITLE
Error when setting the sort order

### DISCUSSION THREAD

**Discussion by 1of2Techs** [microsoft-search/pnp-modern-search#1406] [Q&A] [answered]
When configuring the PNP Results webpart, an error is shown if I use the sort order option.  I've tried several fields.  The fields are in the column list and are marked as sortable.  
[HTTP]:500 - [CorrelationId]:c021f99f-b096-1000-32f4-1976d4c5a5de [Version]:16.0.0.21729
The search works fine if I remove the sort option.

I'm using Chrome on Windows 10.  
Latest version of PNP Search installed.  
This is the first attempt at using the search webpart.
![image](https://user-images.githubusercontent.com/92530842/137372213-a03d7c41-e622-4f45-9743-ca633e746249.png)

**Comment by wobba** [comment, marked as answer]
Fixed in the dev branch and will make it's way to the next release. And logging a bug as a discussion is the wrong approach.
